website/src/pages/covid/single-variant.astro

```diff
@@ -133,6 +133,13 @@ const organismConfig = getDashboardsConfig().dashboards.organisms;
                 views={[views.table, views.bar]}
                 pageSize={10}
             />
+            <GsAggregate
+                title='Sub-lineages Nextstrain Clade'
```

Comment:
```suggestion
                title='Nextstrain Clade'
```

IMO we could also change the title of the other diagram from `Sub-lineage` to `Nextstrain Pango Lineage`